app/views/partials/multi_factor_authentication/_mfa_selection.html.erb

```diff
@@ -19,8 +19,8 @@
     ) do %>
   <%= image_tag(asset_url("mfa-options/#{option.type}.svg"), alt: '', class: 'usa-checkbox__image') %>
   <div class="usa-checkbox__label--text">
-    <%= option.label %>
-    <span id="two_factor_options_form_selection_description_<%= option.type %>" aria-hidden="true">
+    <span class="margin-right-2"><%= option.label %></span>
+    <span id="two_factor_options_form_selection_description_<%= option.type %>" <% if option.mfa_configuration_description.length > 0 %>class="usa-tag bg-green" <% end %>aria-hidden="true">
       <%= option.mfa_configuration_description %>
     </span>
```

Comment:
To make this more readable, I might suggest either breaking this onto a few lines, or switching to a more programmatic tag builder like `content_tag`. Also, I think we could use `.present?` as a more readable/semantic alternative to checking the length of the string.

```suggestion
    <span
      id="two_factor_options_form_selection_description_<%= option.type %>"
      <% if option.mfa_configuration_description.present? %>class="usa-tag bg-green" <% end %>
      aria-hidden="true"
    >
      <%= option.mfa_configuration_description %>
    </span>
```

Or:

```suggestion
    <%= content_tag(
          :span,
          id: "two_factor_options_form_selection_description_<%= option.type %>",
          class: option.mfa_configuration_description.presence && 'usa-tag bg-green',
          aria: { hidden: true },
        ) do %>
      <%= option.mfa_configuration_description %>
    <% end %>
```

Reply:
The `content_tag` snippet is giving me a syntax error but it's not clear to me why.
_Encountered a syntax error while rendering template: check <%= check_box_tag(
      'two_factor_options_form[selection][]',
      option.type,
      false,
      disabled: option.disabled? && option.type === 'piv_cac' || option.disabled? && option.type === 'backup_code' || option.disabled? && option.type === 'phone',_

The previous snippet works great and makes the code cleaner and easier to read.

Reply:
I forgot to swap the interpolation syntax `<%= ... %>` when changing to a string parameter (should be `#{ ... }` instead).

This should work:

```suggestion
    <%= content_tag(
          :span,
          id: "two_factor_options_form_selection_description_#{option.type}",
          class: option.mfa_configuration_description.presence && 'usa-tag bg-green',
          aria: { hidden: true },
        ) do %>
      <%= option.mfa_configuration_description %>
    <% end %>
```